libp2p/Session.cpp

```diff
@@ -292,8 +295,7 @@ void Session::drop(DisconnectReason _reason)
 
 void Session::disconnect(DisconnectReason _reason)
 {
-    cnetdetails << "Disconnecting (our reason: " << reasonOf(_reason) << ") from " << m_info.id
-                << "@" << m_socket->remoteEndpoint();
+    LOG(m_netLoggerDetail) << "Disconnecting (our reason: " << reasonOf(_reason) << ") from";
```

Comment:
Also public method, so we should assume it may be called from any thread

Reply:
Ah okay, I had converted all of these calls because I had done some code inspection and thought they were called from a single thread but you bring up a good point - any public function can be called from any thread....I'll change this log statement back.

Reply:
I think it's not the only change like this in `Session` class, please double-check this